.codecov.yml

```diff
@@ -17,4 +17,3 @@ comment:
   require_changes: false
   require_base: no
   require_head: yes
-
```

Comment:
Minor: We often use newlines in `daily-matrix.yml` to trigger Daily CI builds on a PR. An aggressive formatter means these runs are always going to fail from the formatter check, so this makes it slightly harder to get a green Daily run on a PR. I guess we just need to get in the habit of triggering these builds manually?

Reply:
Or we add an empty daily_canary file which we tell the daily to trigger on but prettier to ignore?